docs/admin/installation-searx.rst

```diff
@@ -64,17 +64,39 @@ from the login (*~/.profile*):
    Open a second terminal for the configuration tasks and left the ``(searx)$``
    terminal open for the tasks below.
 
+
+.. _use_default_settings.yml:
+
 Configuration
-==============
+=============
+
+To create a initial ``/etc/searx/settings.yml`` you can start with a copy of the
+:origin:`searx/settings.yml` or even better with a copy of
+:origin:`utils/templates/etc/searx/use_default_settings.yml`.  The last one is
+recommended since :pull:`2291` is merged, this setup :option:ref:`use default
```

Comment:
The note is inconsistent with the note here : https://searx.github.io/searx/admin/settings.html#use-default-settings

As I understand there are two use cases for the settings:
* a searx clone, with in place modification including settings.yml. In this configuration, there is no need for an additional settings: searx/settings.yml is merged using git.
* a searx clone with the modifications outside the repository as described here https://searx.github.io/searx/admin/installation-searx.html#configuration

Rereading the documentation, the first use case is not documented.

Reply:
> [EDIT] I don't have a clear view on how to document this aspect.

I changed it to:

```rst
To create a initial ``/etc/searx/settings.yml`` you can start with a copy of the
file :origin:`utils/templates/etc/searx/use_default_settings.yml`.  This setup
:option:ref:`use default settings <settings use_default_settings>` from
:origin:`searx/settings.yml` and is recommended since :pull:`2291` is merged.
```